oaknational/Oak-Web-Application
# Feat/ctc 37/update subject listing query unit mv

## Description

update subject listing mv to use published_mv_synthetic_unitvariants_with_lesson_ids_by_keystage_14_0_1


## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-3033--oak-web-application.netlify.thenational.academy
2. Click on subject listing
3. You should see no regressions

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
